src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -1319,17 +1355,44 @@ static int client_receive_message(
 
         case DHCP6_STATE_BOUND:
         case DHCP6_STATE_STOPPED:
+        case DHCP6_STATE_STOPPING:
         default:
                 assert_not_reached();
         }
 
         return 0;
 }
 
+static int client_send_release(sd_dhcp6_client *client) {
+        sd_dhcp6_lease *lease;
+        int r;
+
+        assert(client);
+
+        if (!client->send_release)
+                return 0;
+
+        if (sd_dhcp6_client_get_lease(client, &lease) < 0)
+                return 0;
+
+        if (!lease->ia_na && !lease->ia_pd)
+                return 0;
+
+        client_set_state(client, DHCP6_STATE_STOPPING);
+        r = dhcp6_client_send_message(client);
+        if (r < 0)
+                log_dhcp6_client_errno(client, r,
```

Comment:
After reconsidering, please do not log the failure here, but log in the caller side, and mention that the failure is ignored:
```c
static int client_send_release(sd_dhcp6_client *client) {
        ...
        return dhcp6_client_send_message(client);
}

int sd_dhcp6_client_stop(sd_dhcp6_client *client) {
        ...
        r = client_send_release(client);
        if (r < 0)
                log_dhcp6_client_errno(client, r, "Failed to send to DHCP6 release message, ignoring: %m");
        ...
}
```

Reply:
yes fixed in the last push